api/app/signals/apps/email_integrations/reporter/mail_actions.py

```diff
@@ -9,10 +9,15 @@
 from signals.apps.signals.models import Signal
 from signals.apps.signals.workflow import (
     AFGEHANDELD,
+    AFWACHTING,
+    BEHANDELING,
+    GEANNULEERD,
     GEMELD,
     GESPLITST,
     HEROPEND,
     INGEPLAND,
+    ON_HOLD,
+    VERZOEK_TOT_AFHANDELING,
     VERZOEK_TOT_HEROPENEN
 )
 
```

Comment:
How are mails send for child signals with a parent that is in the state 'GESPLITST'? Because we want 'GESPLITST' to be working as normal, right?